faq.md

```diff
@@ -201,15 +201,29 @@ Flutter is built with C, C++, Dart, and Skia (a 2D rendering engine). See this
 
 ### How does Flutter run my code on Android? {#run-android}
 
-The engine's C/C++ code is compiled with Android's NDK, and any Dart code is AOT-compiled
-into native code. The app runs using the native instruction set (no interpreter
-is involved).
+The engine's C/C++ code is compiled with Android's NDK, and both the SDK's and
```

Comment:
The engine's C and C++ code are compiled with Android's NDK.  The Dart code (both the SDK and your code) are ahead-of-time (AOT) compiled into a native ARM library.